### PR TITLE
Feature/firefly gps

### PR DESCRIPTION
**Pull request by marija-p**
Small addition: I've written a new Firefly URDF file featuring the [hectors GPS plugin] (http://wiki.ros.org/hector_gazebo_plugins). This file is used in mav_control/path_publisher and could also be useful for some projects that need a GPS sensor for simulation. Hopefully, this is in the correct place?

### CODE REVIEW

**Comment by markusachtelik** [comment]
add to whitelist

File: rotors_description/urdf/firefly_base_gps.xacro [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<!--
+  Copyright 2015 Fadri Furrer, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Michael Burri, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Mina Kamel, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Janosch Nikolic, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Markus Achtelik, ASL, ETH Zurich, Switzerland
```
Comment (markusachtelik) [inline rotors_description/urdf/firefly_base_gps.xacro]: add yourself as author ;)

**Comment by markusachtelik** [comment]
looks good to me, thanks! Let's have @ffurrer have a final look.

File: rotors_description/urdf/firefly_base_gps.xacro (hunk 1 above)
Comment (marija-p) [inline rotors_description/urdf/firefly_base_gps.xacro]: Good point :-)

Comment (marija-p) [inline rotors_description/urdf/firefly_base_gps.xacro]: A bit off-topic, but how do I know which copyright template to use for each package?

**Comment by ffurrer** [comment]
test this please.

File: rotors_description/urdf/firefly_base_gps.xacro [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  <!-- Mount an ADIS16448 IMU. -->
+  <xacro:default_imu namespace="${namespace}" parent_link="${namespace}/base_link" />
+
+  <!-- Mount a GNSS receiver. -->
+  <gazebo>
+    <plugin name="${namespace}_gps_sim" filename="libhector_gazebo_ros_gps.so">
+	<alwaysOn>true</alwaysOn>
+	<topicName>fix</topicName>
+	<velocityTopicName>fix_velocity</velocityTopicName>
+	<referenceLatitude>47.3761890018</referenceLatitude>
```
Comment (ffurrer) [inline rotors_description/urdf/firefly_base_gps.xacro]: Can you add `xacro`-properties for this and the values below, e.g. as:

```XML
<xacro:property name="referenceLatitude" value="47.3761890018" />
...
<!-- and then -->
<referenceLatitude>${referenceLatitude}</referenceLatitude>
```

Comment (marija-p) [inline rotors_description/urdf/firefly_base_gps.xacro]: That's been done now. :)

File: rotors_description/urdf/firefly_base_gps.xacro [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  <!-- Instantiate a controller. -->
+  <xacro:controller_plugin_macro namespace="${namespace}" imu_sub_topic="imu" />
+
+  <!-- Mount an ADIS16448 IMU. -->
+  <xacro:default_imu namespace="${namespace}" parent_link="${namespace}/base_link" />
+
+  <!-- Mount a GNSS receiver. -->
+  <gazebo>
+    <plugin name="${namespace}_gps_sim" filename="libhector_gazebo_ros_gps.so">
```
Comment (ffurrer) [inline rotors_description/urdf/firefly_base_gps.xacro]: Hey as this adds a dependency on hector_gazebo, we should probably not add it here but in https://github.com/ethz-asl/rotors_simulator_demos. There I'm fine with adding additional dependencies.
Or what do you think @burrimi @markusachtelik @alexmillane @helenol ?

Comment (marija-p) [inline rotors_description/urdf/firefly_base_gps.xacro]: @ffurrer Okay, makes sense. Let me know if you guys think I should send a pull request there, instead!

Comment (ffurrer) [inline rotors_description/urdf/firefly_base_gps.xacro]: yes, please do so :)

**Comment by marija-p** [comment]
Moved pull request to rotors_simulator_demos based on discussion above.

**Pull request closed without merging**